x-pack/plugins/security_solution/cypress/e2e/detection_alerts/alert_table_hover_actions.cy.ts

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNewRule } from '../../objects/rule';
+import { HOST_NAME, USER_NAME } from '../../screens/alerts';
+import {
+  ALERT_FLYOUT,
+  ALERT_FLYOUT_CLOSE_BTN,
+  CELL_EXPAND_VALUE,
+  CELL_EXPANSION_POPOVER,
+} from '../../screens/alerts_details';
+import {
+  GLOBAL_SEARCH_BAR_FILTER_ITEM,
+  GLOBAL_SEARCH_BAR_FILTER_ITEM_DELETE,
+} from '../../screens/search_bar';
+import {
+  DATA_GRID_FIELDS,
+  GET_DATA_GRID_HEADER,
+  ID_COLUMN_VALUES,
+  TOP_N_ALERT_HISTOGRAM,
+  TOP_N_CONTAINER_CLOSE_BTN,
+} from '../../screens/shared';
+import { HOVER_ACTIONS, TIMELINE_DATA_PROVIDERS_CONTAINER } from '../../screens/timeline';
+import { waitForAlerts, waitForTopNHistogramToLoad } from '../../tasks/alerts';
+import { createCustomRuleEnabled } from '../../tasks/api_calls/rules';
+import { cleanKibana } from '../../tasks/common';
+import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
+import { login, visit } from '../../tasks/login';
+import { addsFieldsToTimeline } from '../../tasks/rule_details';
+import { closeTimelineUsingCloseButton } from '../../tasks/security_main';
+import { openActiveTimeline } from '../../tasks/timeline';
+import { ALERTS_URL } from '../../urls/navigation';
+
+describe('Alerts Table : Hover Actions', () => {
+  before(() => {
+    cleanKibana();
+    login();
+    createCustomRuleEnabled(getNewRule(), 'new custom rule');
+  });
+
+  context('Id Column', { testIsolation: false }, () => {
+    before(() => {
+      visit(ALERTS_URL);
+      waitForAlertsToPopulate();
+      const idColName = DATA_GRID_FIELDS.ID.fieldName;
+      addsFieldsToTimeline(idColName, [idColName]);
+      cy.get(GET_DATA_GRID_HEADER(DATA_GRID_FIELDS.ID.fieldName)).should('be.visible');
+    });
+    beforeEach(() => {
+      /*
+       *
+       * Since now alert table is different from the tables being used
+       * in events, we have to create different set of tests for hover actions
+       * for alerts table.
+       *
+       * These tests will be done on `_id` column since there can be difference
+       * in how `_id` is supplied to the security solution by trigger actions table
+       *
+       * */
+      waitForAlerts();
+      cy.get(ID_COLUMN_VALUES).first().trigger('mouseover', { timeout: 5000 });
+    });
+
+    it('Filter For', () => {
```

Comment:
similar comment I made on Christine [PR](https://github.com/elastic/kibana/pull/151086/files), I feel like the filter in and out tests could be combined into one that test the filter functionality?

Reply:
these 2 tests are failing locally for me btw

cannot find [data-test-subj="filter-for-value"]

Reply:
@PhilippeOberti  yes.. it is because of new PR that has merged. I will resolve conflicts soon.